### PR TITLE
[JBTM-3275] when waiting on server timed-out, we need to stop the server

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.redhat.com/browse/JBTM-3275

A fix for the stuck happened at 
http://narayanaci1.eng.hst.ams2.redhat.com/job/btny-pulls-narayana-quickstart/13/console

### CODE REVIEW

File: XTS/ssl/xts-over-ssh.sh [hunk 1]
```diff
@@ -34,18 +34,19 @@ CURRENT_SCRIPT_DIR=`dirname "$ABSPATH"`
 function waitServerStarted() {
   local PORT=${1:-$CLI_PORT_BASE}
   local RETURN_CODE=-1
-  local TIMEOUT=20 # timeout is 20 seconds
+  local TIMEOUT=40 # timeout is 40 seconds
```
Comment (mayankkunwar) [inline XTS/ssl/xts-over-ssh.sh:37]: @ochaloup Is here any specific reason to change the timeout from 20sec to 40sec?

Comment (ochaloup) [reply XTS/ssl/xts-over-ssh.sh:37]: @mayankkunwar no ;-)
the reason is only the current state of the CI 20 seconds could not be  enough. As normally the WFLY starts in few seconds the timeout to 20s seemed fine, but it fails now.
I was thiking to parametrize the timeout somehow but it should be done on the state of the  all quickstarts to being able to consume some env parameter to multiply the default timeout. But at some places it's a shell script on some places it's arquillian setup, at some places it's java or maven. As every quickstart was written in different time by a different person it's bit difficult. I didn't want to search that now.
Would be fine if I create and issue to investigate on possibilities of timeout parametrizations for quickstart?

Comment (mayankkunwar) [reply XTS/ssl/xts-over-ssh.sh:37]: @ochaloup That would be a good idea to make timeout parameterized in separate PR. For now maybe we can continue with your changes.

Comment (ochaloup) [reply XTS/ssl/xts-over-ssh.sh:37]: Thanks!

**Pull request merged**